README.md

```diff
@@ -76,22 +78,24 @@ Click on `Add notebooks` (upper right) to create a notebook.
 ## 4. Add the data and configuration file
 
 #### Add the data and configuration to the notebook
-Use `Find and Add Data` (look for the `10/01` icon)
-and its `Files` tab. From there you can click
-`browse` and add data and configuration files from your computer.
 
-> Note:  If you don't have your own data and configuration files, you can get our example by cloning
-this git repo. Look in the `data/sample_text.txt` directory for data file and in the `configuration/sample_config.txt`.
-If you use configuration file from your computer, make sure to conform to the JSON structure given in `configuration/sample_config.txt`.
+* From the `My Projects > Default` page, Use `Find and Add Data` (look for the `10/01` icon)
+and its `Files` tab. 
+* Click `browse` and navigate to this repo `watson-document-classifier/data/sample_text.txt
+* Click `browse` and navigate to this repo `watson-document-classifier/configuration/sample_config.txt
 
 ![](doc/source/images/add_file.png)
 
-#### Fix-up variable names
-Once the files have been uploaded into ``DSX-ObjectStore`` you need to update the variables that refer to the data and configuration files in the Jupyter Notebook.
+> Note:  It is possible to use your own data and configuration files.
+If you use a configuration file from your computer, make sure to conform to the JSON structure given in `configuration/sample_config.txt`.
+
+#### Fix-up variable names for your own data and configuration files
```

Comment:
"Fix-up variable names" makes more sense with inserted code like credentials_2 --> credentials_1.  I think this notebook isn't using inserted code here so probably s/Fix-up variable names/Fix-up file names/ or something like that.